tools/postinstall.sh

```diff
@@ -28,10 +28,3 @@ then
   sudo service rsyslog restart
   echo "Finished syslog configuration!"
 fi
-
```

Comment:
in postinstall.sh we have not done any change only we have removed it from install steps. So we are expecting user to run this explicitly right ?

Reply:
Yes